.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run format:check
```

Comment:
Why run only prettier and not pretter+eslint?

Reply:
Eslint is configured to report prettier issues, so running `npm run lint` would take care of both